adal/build.gradle

```diff
@@ -66,6 +68,13 @@ android {
         project.version = android.defaultConfig.versionName
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+    publishing {
+        multipleVariants {
+            allVariants() // Include all variants
```

Comment:
Does this mean all variants are going to be published? If that's the case then I don't think we want that. We only want the release variant to be published